lnst/Controller/Host.py

```diff
@@ -57,16 +57,21 @@ def namespaces(self):
         return ret
 
     def map_device(self, dev_id, how):
-        #TODO if this is supposed to be public it should be better than dict["hwaddr"]!!!!
-        hwaddr = how["hwaddr"]
-        dev = self._machine.get_dev_by_hwaddr(hwaddr)
+        if "hwaddr" in how:
```

Comment:
Why is `how` a dictionary? Why it simply can't be a string? Do you expect the dictionary would have more data in it? If so what are the expected use cases?

A type hint for how would be useful.

Reply:
Sorry .. misunderstood the concept. Resolving.